iml-postgres/src/pool.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+pub use deadpool_postgres::{config::ConfigError, Client, Pool, PoolError};
+
+use iml_manager_env::get_db_conn_hash;
+use tokio_postgres::NoTls;
+
+pub async fn pool() -> Result<Pool, ConfigError> {
```

Comment:
Instead of adding this connection pool, how much work would it be to use sqlx which has built in pooling?

https://github.com/whamcloud/integrated-manager-for-lustre/blob/master/iml-postgres/src/lib.rs#L24-L51